project/plugins.sbt

```diff
@@ -1,10 +1,10 @@
 val scalaJSVersion =
-  Option(System.getenv("SCALAJS_VERSION")).getOrElse("1.2.0")
+  Option(System.getenv("SCALAJS_VERSION")).getOrElse("1.3.1")
```

Comment:
Forced by 3.0.0-M2

Reply:
I *think* this means next version needs to be 0.15.1, but I always get confused about Scala.js bincompat guarantees